### PR TITLE
Add release information and update documentation

### PR DESCRIPTION
**Pull request by feiskyer**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind documentation

**What this PR does / why we need it**:

Add release information and update documentation.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
Add release information and update documentation.
```

### CODE REVIEW

File: docs/cloud-controller-manager.md [hunk 1]
```diff
@@ -33,33 +36,116 @@ To use cloud controller manager, the following components need to be configured:
...
 
+## AzureDisk and AzureFile
+
+AzureDisk and AzureFile volume plugins are not supported with external coud provider (See [kubernetes/kubernetes#71018](https://github.com/kubernetes/kubernetes/issues/71018) for explanations).
+
+Hence, [azuredisk-csi-driver](https://github.com/kubernetes-sigs/azuredisk-csi-driver) and [azurefile-csi-driver](https://github.com/kubernetes-sigs/azurefile-csi-driver) should be used for persistent volumes.
+
+### Deploy AzureDisk CSI plugin
+
+Run following commands:
```
Comment (andyzhangx) [inline docs/cloud-controller-manager.md]: add a deployment link may be more better

Comment (feiskyer) [reply docs/cloud-controller-manager.md]: Added links for each driver.

File: README.md [hunk 2]
```diff
@@ -10,6 +12,18 @@ The Azure cloud provider code locates at [Kubernetes repository directory](https
...
+
+cloud-provider-azure is still under **alpha** stage and its releases are maintained on Microsoft Container Registry (MCR).
+
+The latest version of azure-cloud-controller-manager could be found at `mcr.microsoft.com/k8s/core/azure-cloud-controller-manager:v0.1.0`.
+
+Version matrix:
+
+|Kubernetes version|cloud-provider version|cloud-provider branch|
+|------------------|----------------------|---------------------|
+| v1.14.x          | v0.1.0               | master              |
```
Comment (ritazh) [inline README.md]: For `cloud-provider branch`, should we use `master` or would a tag be better?

Comment (feiskyer) [reply README.md]: We only have master now, after GA, we may have release branches, e.g. release-1.0

**Pull request merged**